docs/.vuepress/config.js

```diff
@@ -30,6 +30,62 @@ module.exports = {
      {
         title: 'About',
         path: '/',
+     },
+     {
+         title: 'Release Notes',
+         path: '/release-notes/',
+         children: [
+           '/release-notes/9.4',
+           '/release-notes/8.9',
+           '/release-notes/8.10-beta',
+            {
+            title: "Older AlmaLinux 9 releases",
+  			  children: [
+              '/release-notes/9.3',
+              '/release-notes/9.2',
+              '/release-notes/9.1',
+              '/release-notes/9.0',
+	            {
+	            title: "AlmaLinux 9 betas",
+	  			  children: [
+                   '/release-notes/9.4-beta',
+				  '/release-notes/9.3-beta',
+				  '/release-notes/9.2-beta',
+				  '/release-notes/9.1-beta',
+				  '/release-notes/9.0-beta',
+	              ]
+	            },
+              ]
+            },
+            {
+            title: "Older AlmaLinux 8 releases",
+  			  children: [
+              '/release-notes/8.8',
+              '/release-notes/8.7',
+              '/release-notes/8.6',
+              '/release-notes/8.5-ppc',
+              '/release-notes/8.5',
+              '/release-notes/8.5-beta-ppc',
```

Comment:
I thinks we should put 8.5-beta-ppc to betas

Reply:
Good catch!